tfhe/src/core_crypto/gpu/slice.rs

```diff
@@ -29,8 +29,8 @@ where
     ///
     /// The ptr must be valid for reads for len * std::mem::size_of::<T> bytes on
     /// the cuda side.
-    pub(crate) unsafe fn new(ptr: CudaPtr, len: usize) -> CudaSlice<'a, T> {
-        CudaSlice {
+    pub(crate) unsafe fn new(ptr: CudaPtr, len: usize) -> Self {
+        Self {
```

Comment:
It looks good to me, but @tmontaigu you will know better than me if this is really ok :slightly_smiling_face:

Reply:
Yes it is